src/app/shared/descriptor-type-compat.service.ts

```diff
@@ -114,8 +114,7 @@ export class DescriptorTypeCompatService {
       case ToolDescriptor.TypeEnum.NFL:
         return 'PLAIN-NFL';
       case ToolDescriptor.TypeEnum.SERVICE:
-        // TODO: Actually handle this
-        return null;
+        return 'PLAIN-SERVICE';
```

Comment:
I rue the day I thought of the `plain` idea

Reply:
Will there ever be a good time to propose a change for it?  or has that ship sailed